android/app/build.gradle

```diff
@@ -207,7 +208,7 @@ dependencies {
     implementation "com.android.support:appcompat-v7:26.1.0"
     // Force using exact RN version instead of relying on gradle dependency resolution
     // https://docs.gradle.org/current/userguide/introduction_dependency_management.html#sec:dependency_resolution
-    compile ("com.facebook.react:react-native:0.53.3") { force = true }  // From node_modules
+    implementation ("com.facebook.react:react-native:0.53.3") { force = true }  // From node_modules
```

Comment:
Why this change?

Reply:
react-native-firebase requires a later version of gradle which requires compile statements to be changed to implementation statemnts

Reply:
@dmitryn WDYT? Looks like it's reverting your recent change?